File: OOP/Circle.js
let Circle = function (radius, color) {
    this.radius = radius;
    this.color = color;
    this.Circle = function () {
        return this;
    };
    this.getRadius = function () {
        return this.radius;
    };
    this.getArea = function () {
        return (Math.PI * radius * radius).toFixed(4);
    };
};
let circle1 = new Circle(12, "red");
console.log(circle1.getArea());
console.log(circle1.getRadius());

File: Module_2/ClassAndObject/Circle/src/main/java/org/example/Circle.java
package org.example;

public class Circle {
    private double radius = 1;
    private String color = "red";
    public Circle(){
    }
    public Circle(double r){
        this.radius = r;
    }

    public double getRadius() {
        return radius;
    }
    public double getArea() {
        return this.getRadius()*this.getRadius()*Math.PI;
    }
}
